jbpm-flow-builder/src/main/java/org/jbpm/osgi/flow/compiler/Activator.java

```diff
@@ -35,7 +36,8 @@ public void start(BundleContext bc) throws Exception {
         this.processBuilderReg = bc.registerService( new String[]{ ProcessBuilderFactoryService.class.getName(), Service.class.getName()},
                                                                    new ProcessBuilderFactoryServiceImpl(),
                                                                    new Hashtable() );
-        ProcessBuilderFactory.reInitializeProvider();
+        RuntimeManagerFactory.Factory.reset();
+        ProcessBuilderFactory.reInitializeProvider();        
```

Comment:
Just thinking out loud -- would it be better to add the call to `jbpm-runtime-manager`'s Activator then?